packages/navigation/assets/index.css

```
@@ -276,3 +276,52 @@
+    align-items: center;
+    justify-content: center;
+}
+/* Visually-hidden accessible text (e.g. the ellipsis "Show more" label). */
+.vc-sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)